kuma/static/js/libs/ckeditor/source/plugins/mdn-sampler/plugin.js

```diff
@@ -32,6 +32,7 @@ CKEDITOR.plugins.add('mdn-sampler', {
         makeElement('pre', gettext('Sample JavaScript Content'), { 'class': 'brush: js' });
 
         // Create the IFrame?
+        makeElement('h3', gettext('Result'));
```

Comment:
Is this [bug 1246684](https://bugzilla.mozilla.org/show_bug.cgi?id=1246684)? If so, please split CKEditor upgrade from the change commit next time there's CKEditor work,